tensorcircuit/translation.py

```diff
@@ -81,6 +82,92 @@ def _merge_extra_qir(
             nqir += inds[k]
     return nqir
 
+class CustomizedCirqGate(cirq.Gate):
+    def __init__(self, uMatrix, name, nqubit):
+        super(CustomizedCirqGate, self)
+        self.uMatrix = uMatrix
+        self.name    = name
+        self.nqubit  = nqubit
+
+    def _num_qubits_(self):
+        return self.nqubit
+
+    def _unitary_(self):
+        return self.uMatrix
+
+    def _circuit_diagram_info_(self, args):
+        return [self.name] * self.nqubit
+
+def qir2cirq(
+    qir: List[Dict[str, Any]], n: int, extra_qir: Optional[List[Dict[str, Any]]] = None
+) -> Any:
+    r"""
+    Generate a cirq circuit using the quantum intermediate
+    representation (qir) in tensorcircuit.
+
+    :Example:
+
+    >>> c = tc.Circuit(2)
+    >>> c.H(1)
+    >>> c.X(1)
+    >>> cisc = tc.translation.qir2cirq(c.to_qir(), 2)
+    >>> print(cisc)
+    1: ───H───X───
+
+    :param qir: The quantum intermediate representation of a circuit.
+    :type qir: List[Dict[str, Any]]
+    :param n: # of qubits
+    :type n: int
+    :param extra_qir: The extra quantum IR of tc circuit including measure and reset on hardware,
+        defaults to None
+    :type extra_qir: Optional[List[Dict[str, Any]]]
+    :return: qiskit cirq object
+    :rtype: Any
+
+    todo:
+    add default theta to iswap gate
+    add more cirq built-in gate instead of customized
+    add unitary test with tolerance
+    add support of cirq built-in ControlledGate for multiplecontroll
+    support more element in qir, e.g. barrier, measure...
+    disable outputting controlled bit when creating controlled gate
```

Comment:
why this is the case, cirq output the info by default?

Reply:
This happens when I use the console to run test_circuit.test_qir2cirq
>> import test_circuit
>> test_circuit.test_qir2cirq(None)
[1, 0]
[0, 1, 0]
[0, 1, 0]

It seems like the tensorcircuit would printout the controlled bit by default, but I haven't checked where this output comes from

Reply:
ah, my bad, have already located the redundency print in tc, will fix together when merge this pr